src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1

```diff
@@ -1612,30 +1613,28 @@ class Organization: ADOSVTBase
                     $controlResult.AddMessage([VerificationResult]::Failed,"Count of inactive guest users in the organization: $($inactiveGuestUsersCount)");
                     $controlResult.AdditionalInfo += "Count of inactive guest users in the organization: " + $inactiveGuestUsersCount;
                     $controlResult.SetStateData("Inactive guest users list: ", $inactiveUsersStateData);
-                    $controlResult.AdditionalInfoInCSV = "GuestUsers: $($users.Count); InactiveGuestUsers: $($inactiveGuestUsersCount); "
+                    $controlResult.AdditionalInfoInCSV = "NumGuests: $($users.Count);"
 
 
                     $inactiveUsersWithDays = $inactiveUsers | Where-Object {$_.InactiveFromDays -ne "User was never active."}
                     $inactiveUsersWithDaysCount = ($inactiveUsersWithDays | Measure-Object).Count
                     if($inactiveUsersWithDaysCount -gt 0) {
                         $controlResult.AddMessage("`nCount of guest users who are inactive from last $($GuestUserInactivePeriodInDays) days: $($inactiveUsersWithDaysCount)");
-                        $inactiveUsersTable = ($inactiveUsersWithDays | FT | Out-String)
+                        $inactiveUsersTable = ($inactiveUsersWithDays | Sort-Object InactiveFromDays -Descending | FT  | Out-String)
                         $controlResult.AddMessage("Inactive guest users list: `n$inactiveUsersTable");
                         $controlResult.AdditionalInfo += "Count of guest users who are inactive from last $($GuestUserInactivePeriodInDays) days: " + $inactiveUsersWithDaysCount;
                     }
-
-                    $domainsInfo = $inactiveguestUsers | ForEach-Object { $_.user.mailaddress.Split("@")[1]} | select-object -Unique -First 10
-                    $controlResult.AdditionalInfoInCSV  += "DifferentDomains: $($domainsInfo -join ', ')"
+                    
```

Comment:
domains data which was available earlier should be added to additional info

"First 10 domains of inactive guest users:"